test/test_errors.py

```diff
@@ -85,10 +85,10 @@ def test_handle_result_error(container_factory, rabbit_config):
             # proxy.task() will hang forever because it generates an error
             proxy.task.call_async()
 
-        with eventlet.Timeout(1):
```

Comment:
I don't think we should put them back. We use `pytest-timeout` to catch hanging tests now, which is what these were originally used for.

Tracking the average test execution time would be a much better way to spot performance issues. It probably wouldn't be too difficult to send stats to a free tier on something like https://plot.ly.